src/RolloutScorer/RolloutScorer/Commands.cs

```diff
@@ -62,30 +62,30 @@ public async Task<int> InvokeAsync(IEnumerable<string> arguments)
             if (string.IsNullOrEmpty(_rolloutScorer.OutputFile))
             {
                 _rolloutScorer.OutputFile = Path.Combine(Directory.GetCurrentDirectory(),
-                    $"{_rolloutScorer.Repo}-{_rolloutScorer.RolloutStartDate.Date.ToShortDateString().Replace("/","-")}-scorecard.csv");
+                    $"{_rolloutScorer.Repo}-{_rolloutScorer.RolloutStartDate?.Date.ToShortDateString().Replace("/","-")}-scorecard.csv");
             }
 
             _rolloutScorer.RolloutWeightConfig = StandardConfig.DefaultConfig.RolloutWeightConfig;
             _rolloutScorer.GithubConfig = StandardConfig.DefaultConfig.GithubConfig;
 
             // If they haven't told us to upload but they also haven't specified a repo & rollout start date, we need to throw
-            if (string.IsNullOrEmpty(_rolloutScorer.Repo))
+            if (string.IsNullOrEmpty(_rolloutScorer.Repo) || _rolloutScorer.RolloutStartDate is null)
```

Comment:
I'd personall have done "!thing.HasValue".  "is null" might have special handling for Nullalbe though?  It's weird that a thing that _very much is not null_ is somehow "is null".  Oh C#...

Reply:
Yeah, I think it's just ol' fogie me not liking new things. :-) It's mind bending that a thing that is clearly a value type (and there for cannot be null) "is null".  Like the fact I can do "!(thing is null)" then call "thing.GetValueOrDefault()" and that doesn't crash hurts my head. :-)